src/webprojects/fxa.md

```diff
@@ -0,0 +1,116 @@
+* Translation is of good quality and makes sense in context.
+* No broken UI, especially in a mobile environment. Text wraps or line breaks correctly.
+* Terms are consistent between pages.
+* Basic font support.
+* Error and warning messages.
+
+#### Functionality testing
+
+* Configure the Settings. Go through all of the categories.
+* Check for email notifications with each of the initial setups.
+* Make changes to some of the settings, check for possible error messages or warnings; check for followup confirmation emails.
+
+### Testing the subscription platform
+
+The subscription platform (or sub plat) consists of a few features. Right now only the _Payment and subscription creation_ is available for testing:
+
+* _Payment and subscription creation_: The initial payment funnel where users enter their credit card information to subscribe to a product. Mozilla partners with Stripe to handle this.
+* _Billing_: Ongoing billing and receipt emails once a subscription is entered.
+* _Subscription management UI_: Settings that allow the user to update their payment information.
+
+Payment testing is tied to the language of user preference and location. Payment service is market specific. For instance, the product is available in German, but the service is available in Germany but not in Switzerland.
+
+#### Before testing
+
+* Have a Firefox account. If you don’t have one, the staging link will take you to the signup process.
+* Choose a Firefox browser for: desktop (Windows only), Android, or iOS. Support for macOS and Linux is coming soon.
+* If you are not already using the browser in the language you are going to test in, type in the URL field `about:Preferences`. Or follow the steps detailed in this [SUMO article](https://support.mozilla.org/en-US/kb/choose-display-languages-multilingual-web-pages) to configure to the correct display language.
+* Have the staging link ready for [subcription testing](accounts.stage.mozaws.net/subscriptions/products/prod_HEJ13uxjG4Rj6L?plan=plan_HEJ1l12p1taV7I)
+
+**Tips:** Feeling not having enough test accounts to use for testing? You are not alone. Here are some tips you can consider. Note that this works with Gmail and Google Apps domains:
+
+* Add number(s) or letter(s) to your existing account to create more accounts by adding a `+` in your email address. For example, if your main email is `name@xyz.com`, your expanded accounts could be `name+1@xyz.com` and `name+m@xyz.com`. You will receive email notifications in your `name@xyz.com` inbox. You can add anything you want, several digits and/or letters.
+* Keep your password simple so you can log in and log out easily. Keep track of which account is connected with which password. Of course, there is Lockwise for the rescue!
+
+#### Testing payment and subscription creation
+
+* Go to the [subscription staging server](accounts.stage.mozaws.net/subscriptions/products/prod_HEJ13uxjG4Rj6L?plan=plan_HEJ1l12p1taV7I)
```

Comment:
Broken link, missing https://

Also, this is not the "subscription staging server", this is a link to a test subscription.